### ISSUE TITLE
Encounter Loss Plateau when training Qwen3VL-2B-Instruct on MMEB-train Datasets

### ISSUE THREAD

**Issue by generalfyl** [QwenLM/Qwen3-VL-Embedding#42]
Hi, This is an amazing work! 

I've already started to use Qwen3VL-2B-Instruct to train an embedding model on the MMEB-train dataset before you released your own embedding models. While the **final downstream performance on MMEB-Eval (Image Only) Benchmark looks reasonable**, I encountered a strange phenomenon during training and would like to ask whether this is expected or has been observed by your team.

### Problem Description

When using Qwen3VL-2B-Instruct as the base model, the training loss shows a clear **Loss Plateau** in the early stage: During approximately the first 200+ training steps, The loss across all MMEB sub-datasets remains almost flat and shows little to no decrease. The gradient norm is extremely small during this phase. After this plateau stage, the loss suddenly starts to drop sharply and training proceeds normally. Despite this behavior, the final evaluation scores on MMEB Eval benchmark are normal and comparable to expectations.

### Experiments and Observations

To rule out optimization-related issues, I conducted multiple controlled experiments: Tried several learning rates and different warmup step configurations. All other training settings and framework remain unchanged.

However, none of these changes eliminated the early-stage loss plateau.

As a comparison: Using the exact same training framework, data, and hyper-parameters, but **switching the base model to Qwen2VL-2B-Instruc**t, The training loss decreases smoothly from the very beginning, And no loss plateau or extremely small Gradnorm issue is observed.

### Questions

1. Have you observed a similar early-stage loss plateau when training embedding or contrastive models based on Qwen3VL-2B-Instruct?
If so, is this behavior related to:  Model initialization?  Pretraining or instruction tuning characteristics?  Architectural or normalization differences between Qwen2VL and Qwen3VL?

2. If you encountered same problem. Is this plateau a known or expected characteristic of Qwen3VL-2B-Instruct, or something that should be mitigated during downstream training? (Since I use Qwen2VL-2B as base model didn't observe this phenomenon at all)

### Loss Curves

<img width="2230" height="1424" alt="Image" src="https://github.com/user-attachments/assets/11f18160-0041-4095-8bf0-be19b6a4f985" />

<img width="2252" height="1402" alt="Image" src="https://github.com/user-attachments/assets/a1895461-5833-46e0-879d-62c978400d21" />

<img width="2220" height="1434" alt="Image" src="https://github.com/user-attachments/assets/ea8116c1-1915-48b7-bf72-cfa11cccd172" />

Any insights or suggestions would be greatly appreciated.
Thank you very much for your excellent work and for maintaining this project!

**Comment by 8igfive** [comment]
Hello, we have also observed the phenomenon you described in our experiments.

* First, regarding your second question: since this phenomenon (seemingly) does not affect the model's final results, we did not investigate or address it more deeply.

* Now for your first question: we observed this phenomenon across multiple architectures (qwen2.5-vl, qwen3-vl), multiple sizes (2B, 4B, 8B), and different training datasets. However, this phenomenon only occurs when using instruct models for representation pretraining. When using the first-stage model for subsequent stage training, this phenomenon is not observed. Therefore, my hypothesis is that this is related to the instruct model being initially unsuitable for representation tasks, combined with weak learning signals in the early stages (including factors such as low initial learning rate and optimizer momentum not yet being accumulated), which prevents the loss from decreasing.

**Comment by generalfyl** [comment]
> Hello, we have also observed the phenomenon you described in our experiments.
> 
> * First, regarding your second question: since this phenomenon (seemingly) does not affect the model's final results, we did not investigate or address it more deeply.
> * Now for your first question: we observed this phenomenon across multiple architectures (qwen2.5-vl, qwen3-vl), multiple sizes (2B, 4B, 8B), and different training datasets. However, this phenomenon only occurs when using instruct models for representation pretraining. When using the first-stage model for subsequent stage training, this phenomenon is not observed. Therefore, my hypothesis is that this is related to the instruct model being initially unsuitable for representation tasks, combined with weak learning signals in the early stages (including factors such as low initial learning rate and optimizer momentum not yet being accumulated), which prevents the loss from decreasing.

Thanks for your answer!

**Comment by kaichengyang0828** [comment]
@generalfyl Hi, i encountered a similar issue. Could you tell me what the global batch size was for your loss curve?
